### DISCUSSION TITLE
ProcessingError thrown when syncing tables with large unit amounts

### DISCUSSION THREAD

**Discussion by hjhdaniel** [codatio/codat-docs#453] [General]
Hi all,

Recently, I found an issue with Codat sync:
`ProcessingError` will be thrown during the syncing of data to Codat if the data contains unit numbers that are approximately `>10 billion`.

I have opened a ticket with the Codat team and they are currently working on the issue (no timeline provided for the fix yet). Would just like to open a thread here so that the issue can be updated by the Codat team here and discussed/monitored by anybody who may be affected by it.

cc: @jmyorston

**Comment by dcoplowe** [comment]
Hi Daniel,

Sorry for the delay. We are looking into this and one of my colleagues has been in contact with you.

Hopefully, we should be able to get this sorted for you soon.

Many thanks,

David
